stock-plugins/aiverify.stock.partial-dependence-plot/algorithms/partial_dependence_plot/aiverify_partial_dependence_plot/__main__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Allow aiverify_partial_dependence_plot to be executable through
+`python3 -m aiverify_partial_dependence_plot`
+"""
+import sys
+from importlib.metadata import version
+from pathlib import Path
+
+from aiverify_partial_dependence_plot.plugin_init import run
+
+
+def main() -> None:
+    """
+    Print the version of test engine core
+    """
+    print("*" * 20)
+    print(version_msg())
+    print("*" * 20)
+    # invoke algorithm
+    run()
+
+
+def version_msg():
+    """
+    Return the aiverify_partial_dependence_plot version, location and Python powering it.
+    """
+    python_version = sys.version
+    location = Path(__file__).resolve().parent.parent
+
+    return (
+        f"Partial dependence plot - {version} from {location} (Python {python_version})"
```

Comment:
This returns: `Partial dependence plot - <function version at 0x10082c430> from ...`. That's because version is a function and you need to call it on a package e.g.
```py
from importlib.metadata import version  
version('wheel') 
```

I might have missed this out when reviewing the other plugins as well and if so you can patch it in the next plugin update